loadtest/store/memstore/store.go

```diff
@@ -103,3 +103,8 @@ func (s *MemStore) SetChannelMember(channelId string, channelMember *model.Chann
 func (s *MemStore) ChannelMember(channelId, userId string) (*model.ChannelMember, error) {
 	return s.channelMembers[channelId][userId], nil
 }
+
+func (s *MemStore) RemoveChannelMember(channelId string, userID string) error {
```

Comment:
Actually, no. :P Idiomatic Go is to make it `userID` Check: https://github.com/golang/go/wiki/CodeReviewComments#initialisms.

There is even a golint check for it in the server repo. :)

Reply:
Yes, I'm getting this lint warnings/suggestions about to change the name hahaha

Reply:
In the past when this has come up we've decided to stick with the `userId` format for consistency - it would have been better if we'd done `userID` all along, but we didn't. It's not really viable to change it e.g. in the `model` package of MM so better to be consistent across the board, even if it isn't the go "standard".

Reply:
> Actually, no. :P Idiomatic Go is to make it `userID` Check: https://github.com/golang/go/wiki/CodeReviewComments#initialisms.
> 
> There is even a golint check for it in the server repo. :)

This is just about keeping consistency with the codebase, not personal preference.

Reply:
My apologies. When you said "OCD stuff", I misunderstood and thought that it was a personal preference.

And FWIW, we are actually changing some `Id`s to `ID` here - https://github.com/mattermost/mattermost-server/pull/13090/files.